homework_3/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Homework_3</title>
+	<link rel="stylesheet" href="css/reset_css.css">
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<div class="box">
+		<img class="logo" src="img/logo.png" alt="logo">
+		<ul class="contacts">
```

Comment:
probably, it was better to contain logo and contacts block into header element and set to it clearfix class